index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Portfolio Setup and Mobile first</title>
+  </head>
+  <body>
+    <header class="logo-menu">
+      <div class="logo" onclick="window.location.href='index.html'">
+        <a href="index.html">My Logo</a>
+      </div>
+      <nav class="fa fa-bars bars"></nav>
+    </header>
```

Comment:
- [ ] [OPTIONAL] Good job so far! The `logo` (if the design has one) should be on the left and has an `a` link (`a` tag). This is for three reasons: `convention`, `semantics`, and `user` experience(`UX`). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info and the `hamburger` button should be on the `right` and has a `nav` tag. The `nav` tag is for the `semantics` reason and the right position for convention and user experience for mobile devices. Please keep in your mind my suggestions 🙏🏼 


```HTML
<header>
    <a href...>Website Logo</a>
    <nav>
        <!-- Main Navigation inside <header> -->
        <div>Hamburger Btn</div>
        <ul>
            <li></li>
        </ul>
    </nav>
</header>
```

Reply:
noted